### PR TITLE
Chaudhryma

### PR DESCRIPTION
**Pull request by Mariachaudhry**

### CODE REVIEW

**Review by alenaminsk — APPROVED** [review]

File: src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package ServiceNow.NERD.Pages;
 
 import java.util.List;
+
```
Comment (andreyeve) [inline src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java:4]: Maria, the extra space can be removed here. Also, there were some other classes where I saw extra spaces, please review all of them to make sure we don't have any. Thank you!

File: src/test/java/ServiceNow/NERD/StepsImplementation/NERDApplicationStepsImplementation.java
Comment (andreyeve) [inline src/test/java/ServiceNow/NERD/StepsImplementation/NERDApplicationStepsImplementation.java:1]: I know that Diego is against all the sleeps unless they are absolutely necessary:) Can you please check if you need them all here?

File: src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Feature: Covid-19 Activities Submission page Scenerios
+  Description: This feature file contains Covid-19 Activities Submission page scenerios
+
+  @SS-3419 @SS-3449 @inProgress
+  Scenario: Verifying additional question on COVID submission and entry form - Service Portal
+#    Given a Program Staff member is on the CRS Knowledge Management System "Submissions" page
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature:6]: Commented Step should not be pushed to master if it is no longer used.

**Review by georgeb79 — APPROVED** [review]

File: src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Feature: Covid-19 Activities Submission page Scenerios
+  Description: This feature file contains Covid-19 Activities Submission page scenerios
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature:2]: The purpose of the Feature keyword is to provide a high-level description of a feature, and to group related scenarios. I am not why do we have a separate line for Description if it all can be included within Feature.

File: src/test/java/ServiceNow/NERD/Features/NCI_Staff_Members.feature [hunk 4]
```diff
@@ -1,9 +1,9 @@
 Feature: NCI Staff Member Test Scenarios
-  
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/NERD/Features/NCI_Staff_Members.feature:2]: Empty line

**Review by hamidnesar — COMMENTED** [review]
The purpose of the Feature keyword is to provide a high-level description of a feature, and to group related scenarios. I am not why do we have a separate line for Description if it all can be included within Feature.

File: src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+  @SS-3419 @SS-3449 @inProgress
```
Comment (zamant2) [inline src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature:4]: @inProgress tag shouldn't merge to master and please remove commented Line#6

File: src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java [hunk 6]
```diff
@@ -242,8 +251,18 @@ public class CreateNewSubmissionPage extends CommonUtils {
...
+	 * @param authorText
+	 * @return
+	 */
+	public static WebElement authorText(String authorText) {
+		return WebDriverUtils.webDriver.findElement(By.xpath("//div[contains(text(),'" + authorText + "')]"));
+	}
+
 	/*** CRS Knowledge Management System "AutomationTest" Under Review Text */
-	@FindBy(xpath = "//a[contains(text(),'AutomationTest')]/following::div[2]")
+	@FindBy(xpath = "///md-list-item[1]//div[2]//div[1]")
```
Comment (zamant2) [inline src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java:265]: is this xpath working? seeing 3 slashes

File: src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java (hunk 1 above)
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java]: Resolved

**Review by sarwarahmed1 — APPROVED** [review]

File: src/test/java/ServiceNow/NERD/StepsImplementation/NERDApplicationStepsImplementation.java
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/NERD/StepsImplementation/NERDApplicationStepsImplementation.java:1]: This issue is under discussion with Diego for NERD. I will try to resolve this by the next PR. Thanks

File: src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature (hunk 3 above)
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature:2]: deleted.

File: src/test/java/ServiceNow/NERD/Features/NCI_Staff_Members.feature (hunk 4 above)
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/NERD/Features/NCI_Staff_Members.feature:2]: resolved

File: src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature (hunk 5 above)
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/NERD/Features/Covid-19ActivitiesSubmissions.feature:4]: resolved!

File: src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java (hunk 6 above)
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java:265]: resolved.

Comment (zamant2) [reply src/test/java/ServiceNow/NERD/Pages/CreateNewSubmissionPage.java]: Still seeing the same xpath, can you check if this xpath is working?

File: src/test/java/ServiceNow/NERD/Features/CRS_Reviewers.feature [hunk 7]
```diff
@@ -1,5 +1,4 @@
-Feature: CRS Reviewers Test Scenarios 
-
+Feature: CRS Reviewers Test Scenarios
 Description: This feature file contains test scenarios related to CRS Reviewers
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/NERD/Features/CRS_Reviewers.feature:2]: Could also delete Description and add it to the Feature keyword if necessary.

**Review by zamant2 — APPROVED** [review]

File: src/test/java/ServiceNow/NERD/Features/CRS_Reviewers.feature (hunk 7 above)
Comment (Mariachaudhry) [reply src/test/java/ServiceNow/NERD/Features/CRS_Reviewers.feature]: resolved

**Review by hamidnesar — APPROVED** [review]

**Review by andreyeve — APPROVED** [review]

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - Juarezds

**Pull request merged**